ports/esp32/mpconfigport.h

```diff
@@ -144,7 +144,9 @@
 #define MICROPY_PY_ONEWIRE                  (1)
 #define MICROPY_PY_SOCKET_EVENTS            (MICROPY_PY_WEBREPL)
 #define MICROPY_PY_BLUETOOTH_RANDOM_ADDR    (1)
+#ifndef MICROPY_PY_BLUETOOTH_DEFAULT_GAP_NAME
 #define MICROPY_PY_BLUETOOTH_DEFAULT_GAP_NAME ("ESP32")
+#endif
```

Comment:
Maybe we can just remove this define altogether, and then it gets the generic default defined in `extmod/nimble/modbluetooth_nimble.c`?

Reply:
I see the esp32 port is the only port that changes the default name; having it be at the default indeed seems fine. I'll push an update.